frontend/src/components/common/ToastMessage.vue

```diff
@@ -58,7 +58,7 @@ const onError = (error: any, info: string) => {
             );
         } else if (status == 409) {
             useNotificationMessage.isNotificationVisible = false;
-            useErrorDialog.dialogTitle = axiosResponse.statusText;
+            useErrorDialog.dialogTitle = axiosResponse.statusText || 'Error';
```

Comment:
Previous code (which I didn't know why) should not have used Axios' "**statusText**" for displaying to user on Dialog component. This is not a message/code/title from our backend, is purely Axio's http verbose, and probably depends on the library version.
I feel this **axiosResponse.statusText** should be removed and for now may just gave it a general suitable title based on what status it encounters.

Reply:
I'm actually not so sure, cause when we run our code locally, it returns the statusText. But after deploy to AWS, it doesn't show anymore.